.gitbook/5-oracles/1-operate-oracle-nodes/2-oracle-intialization.md

```diff
@@ -0,0 +1,144 @@
+# Initialization
+
+#### Before run the oracle, you should initialize the oracle configuration.
+
+## Preparations
+
+Before initializing the oracle, you should install the oracle described
+in [oracle-installation](./1-oracle-installation.md).
+After install the oracle, create an empty directory on your host, to be mounted as the `/home_mnt` directory in the
+enclave.
+
+```bash
+# If you run oracle on your host
+sudo mkdir /oracle
+
```

Comment:
I think we could just delete the examples that don't use docker. 
Users will be confused.
wdyt?

Reply:
Applied in [2f0e3b3](https://github.com/medibloc/panacea-core/pull/611/commits/2f0e3b315aac2c1cebfc7a7cbda8ba0faea59a51).